include/zephyr/sw_isr_table.h

```diff
@@ -64,6 +64,12 @@ struct _isr_list {
 /** This interrupt gets put directly in the vector table */
 #define ISR_FLAG_DIRECT BIT(0)
 
+/**
+ * This interrupt gets put directly in the vector table and can
+ * be shared with up to three ISRs
```

Comment:
The Kconfig file states `This option allows up to 4 ISRs to share 1 IRQ line` ... presumably this should be four (`MAX_SHARED_ISRS = 4`).

Reply:
I've removed the max number of shared ISRs. Any number of ISRs can been shared.